src/os/bluestore/BlueStore.cc

```diff
@@ -3760,6 +3760,11 @@ int BlueStore::_open_alloc()
   alloc = Allocator::create(cct, cct->_conf->bluestore_allocator,
                             bdev->get_size(),
                             min_alloc_size);
+  if (alloc == nullptr) {
+    lderr(cct) << "Allocator::unknown alloc type" << cct->_conf->bluestore_allocator << dendl;
```

Comment:
and you can more specific in your commit message title, "fix bug" is too vague a wording.